tendrl-frontend.spec

```diff
@@ -1,12 +1,12 @@
-Name: tendrl-frontend
+Name: dashboard
```

Comment:
- rpm names require product prefix. Make this "tendrl-dashboard"
- Rename this file to "tendrl-dashboard.spec"